etc/rc.init

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
I think shebang won't be recognized if there are preceding white spaces in a file.